frontend/src/components/MapView/Legends/index.tsx

```diff
@@ -145,15 +165,24 @@ function Legends({ classes, layers, extent }: LegendsProps) {
             )}
             <Divider />
             <Grid item>
-              <Button
-                variant="contained"
-                color="primary"
-                size="small"
-                onClick={e => handleAnalysisDownload(e)}
-                fullWidth
-              >
-                {t('Download')}
-              </Button>
+              <MultiOptionsButton
+                mainLabel="Download"
+                options={[
+                  {
+                    label: 'GEOJSON',
+                    onClick: handleAnalysisDownloadGeoJson,
+                  },
+                  {
+                    label: 'CSV',
+                    onClick: handleAnalysisDownloadCsv,
+                    disabled: !(
```

Comment:
Does this mean that it will be shown but greyed out? Maybe we should hide it altogether instead? @wadhwamatic what do you think?

Reply:
Yes it will be greyed out for now

Reply:
Just pushed a commit to hide the option